src/main/camhook/cam.h

```diff
@@ -3,6 +3,13 @@
 
 #include "camhook/config-cam.h"
 
+enum camhook_version {
+    CAMHOOK_OLD,
+    CAMHOOK_NEW,
+};
```

Comment:
question: What's old vs. new and why keep the old "thing"?

Reply:
it controls the vid/pid used, needs to be kept else old support breaks